docs/developers_guide.md

```diff
@@ -1,108 +1,101 @@
-# Developers Guide
+# Developer's Guide
 
 ## Tech preview
 
 ### Files
 
-The repository has below logic files. The every logic is included in the `install.py`. The reason is to run it directory from `curl` command without `pip`. I think that is useful to debug before actual release. But it makes the source code hard to read.
-If we can compress the logic from .py files to one py file including zipped binary code like [get-pip.py](https://github.com/pypa/get-pip/blob/master/get-pip.py), we can split the `install.py` to several py files.
+The repository has the following logic files:
 
 * setup.py
 * install.py
 
-and below testing files.
+All logic is included in `install.py`. The reason for that is to be able to run it directly via the `curl` command without `pip`. I think that is useful to debugging before an actual release. But it makes the source code hard to read.
+If we can compress the logic from `.py` files to one `.py` file including zipped binary code like [get-pip.py](https://github.com/pypa/get-pip/blob/master/get-pip.py), we can split `install.py` to several `*.py` files.
 
-* tests/test_install.py
-* tests/test_integration.py
+The following files include the tests:
+
+* [tests/test_install.py](../tests/test_install.py)
+* [tests/test_version.py](../tests/test_version.py)
+* [tests/test_integration.py](../tests/test_integration.py)
+* [tests/test_install_fedora.py](../tests/test_install_fedora.py)
+* [tests/test_install_suse.py](../tests/test_install_suse.py)
```

Comment:
Why `../` in the relative path for the link? `[tests/test_install.py](tests/test_install.py)` without `../` is better and enough isn't it?

Reply:
Here are the cases not using "../".

```
$ grep -n docs README.md 
108:See [User's guide](docs/users_guide.md).
114:See the [Developers guide](docs/developers_guide.md) for further details.
```

Reply:
> Why `../` in the relative path for the link? `[tests/test_install.py](tests/test_install.py)` without `../` is better and enough isn't it?

The advantage of that is, that my editor will automatically open the correct file if I put the cursor on `../tests/test_install.py` and press the key-chord for open-file (as that is the correct relative path). In contrast to that `/tests/test_install.py` will be interpreted as an absolute path starting at the filesystem root (by my editor of course only, Github doesn't care).

It is a small convenience, but not really crucial so I don't insist on that.

Reply:
> It is a small convenience, but not really crucial so I don't insist on that.

I see. I want you to remove the `../`. Because I checked popular repositories ruby and rails that had `README.md`. They do not use `../` format for the links, and also for a consistency with other parts in this repository's md files.

```
* [tests/test_install.py](tests/test_install.py)
* [tests/test_version.py](tests/test_version.py)
* [tests/test_integration.py](tests/test_integration.py)
* [tests/test_install_fedora.py](tests/test_install_fedora.py)
* [tests/test_install_suse.py](tests/test_install_suse.py)
```

Reply:
Reverted back.

Reply:
Thanks!
But I would prefer not putting the heading `/` for the pats.

Below is your modifications. But I think it's better `/tests/test_install.py` URL is changed to `tests/test_install.py` deleting the heading `\`, because of consistency with other parts.

```
+* [tests/test_install.py](/tests/test_install.py)
+* [tests/test_version.py](/tests/test_version.py)
+* [tests/test_integration.py](/tests/test_integration.py)
+* [tests/test_install_fedora.py](/tests/test_install_fedora.py)
+* [tests/test_install_suse.py](/tests/test_install_suse.py)
```

Could you change it again?
Thanks!

Reply:
I've just tried this and that breaks the linking, because gfm interprets a `(tests/test_install.py)` as a **relative** link with respect to the current location, i.e. to `docs/tests/test_install.py`, which does not exist, so I've changed it back to `/tests/`. Please keep in mind that the part in ordinary braces is **not** visible on GitHub, it is merely used to create a link. The part that is used to generate the link text is in `[]`.

Reply:
Ah I misunderstood it. I am really sorry for that. `docs/developers_guide.md` is at `docs` directory. I misunderstood `developers_guide.md` was at top directory.

So, `../tests/test_install.py` was the correct relative path.
Could you change `/tests/test_*.py` to `../tests/test_*.py` again?
Sorry. If you do not like the change again, I will change it after merging this PR.

Reply:
No problem, I've changed it to use relative paths.